Foundation/FileManager.swift

```diff
@@ -82,6 +96,8 @@ open class FileManager : NSObject {
             }
             urls = mountPoints(statBuf, Int(fsCount))
         }
+#elseif os(Windows)
+        return nil
```

Comment:
Bleh, this seems wrong.  This should be changed to `throw NSUnimplemented` at the very least